Source/doom.cpp

```diff
@@ -67,10 +64,11 @@ void doom_cleanup()
 }
 
 #ifdef HELLFIRE
-BOOLEAN doom_load_graphics()
+BOOLEAN
 #else
-void doom_load_graphics()
+void
 #endif
+    doom_load_graphics()

```

Comment:
It takes up more lines and is harder to read, imo nothing is saved here.

Reply:
that's how other functions were done, I just adjusted mine to match their format 🤷‍♀

Reply:
What other functions?

Reply:
SyncPutItem